target/linux/ramips/image/Makefile

```diff
@@ -56,6 +57,22 @@ define Build/loader-kernel
 	$(call Build/loader-common,LOADER_DATA="$@")
 endef
 
+define Build/loader-common-sercomm
+	rm -rf $@.src
+	$(MAKE) -C lzma-loader \
+		PKG_BUILD_DIR="$@.src" \
+		TARGET_DIR="$(dir $@)" LOADER_NAME="$(notdir $@)" \
+		BOARD="$(BOARDNAME)" PLATFORM="$(LOADER_PLATFORM)" \
```

Comment:
$(BOARDNAME) ? Does this still exist for the devices in this target?

Reply:
This exists and contains the following: BOARD="**MT7621 based boards**".

Reply:
So, it looks like it's not doing anything useful ...

Reply:
Would you like me to remove it and test?

Reply:
I would keep what's in the original implementation, and I think it's there as well ...